cmd/common/display/message.go

```diff
@@ -107,16 +107,18 @@ func (r *RespTxQuery) MarshalJSON() ([]byte, error) {
 		TxResult: r.Msg.TxResult,
 	}
 	// Always try to serialize to verify hash, but only show raw if requested.
-	raw, err := r.Msg.Tx.MarshalBinary()
```

Comment:
Pretty wild that this doesn't error with `r.Msg.Tx == nil`, but it's just doing `serialize.Encode(nil)` which RLP is just fine with  :facepalm: